Map: update demo descriptions (#28395)
DevExpress/DevExtreme: apps/demos/Demos/Map/ProvidersAndTypes/description.md

The [provider](/Documentation/ApiReference/UI_Components/dxMap/Configuration/#provider) property specifies which map provider to use: Google Maps (default), Azure Maps, Google Static Maps, or Bing Maps (deprecated).

You can also set the [type](/Documentation/ApiReference/UI_Components/dxMap/Configuration/#type) property to specify the map type: road, satellite (photographic), or hybrid map.
<!--split-->

[note]

On May 21, 2024, Microsoft announced that [Bing Maps for Enterprise and its API will be discontinued](https://www.microsoft.com/en-us/maps/bing-maps/discontinued-services). [Azure Maps](https://azure.microsoft.com/en-us/products/azure-maps/) will be a single unified enterprise mapping platform available from Microsoft.

> If you have an existing license to Bing Maps for Enterprise, you can continue using Bing provider for our Map component until the end of its support. You need to transition to new API until June 30, 2025 (free and basic license) or until June 30, 2028 (enterprise license).

Reviewer: ```suggestion
If you have an existing Bing Maps for Enterprise license, you can continue using Bing for our Map component until its support ends. You need to transition to the new API before June 30, 2025 (free and basic license) or before June 30, 2028 (enterprise license).
```

> The last date you can get a new license to Bing Maps for Enterprise is June 30, 2024. If you do not have an existing license after that date, you can only use other [map providers](/Documentation/ApiReference/UI_Components/dxMap/Types/#MapProvider) supported by our controls: Azure, Google, and Google Static.

Reviewer: ```suggestion
The last date you can get a new Bing Maps for Enterprise license is June 30, 2024. If you do not have an existing license after this date, you can only use other [map providers](/Documentation/ApiReference/UI_Components/dxMap/Types/#MapProvider) supported by our controls: Azure, Google, and Google Static.
```
